webapp-django/crashstats/crashstats/models.py

```diff
@@ -802,15 +767,12 @@ class RawCrash(SocorroMiddleware):
         "AvailablePhysicalMemory",
         "AvailableVirtualMemory",
         "BIOS_Manufacturer",
-        "bug836263-size",
-        "buildid",
         "BuildID",
         "CrashTime",
         "CoUnmarshalInterfaceResult",
         "DOMIPCEnabled",
         "EMCheckCompatibility",
         "FlashVersion",
-        "id",
```

Comment:
I think this used to be ProductID a long time ago or something like that. It's not sent anymore.